.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+/dist
```

Comment:
I have pointed out this issue many times and asked you to self-review it on GitHub after raising the PR. 
 - No new line at the end of the file.

Reply:
Sir, If you mean I should add new Line at the end of the file , I did If I did not add the new line at the end GitHub Would Have showed a red marker at the end